### ISSUE TITLE
Elapsed Time is always 16 ms

### ISSUE THREAD

**Issue by galadran** [AlanWills/Other-Worlds-Than-These#2]
I don't know if this is failure in my debugging, maybe I'm doing something wrong. But in the SpaceScreen / AddRandomEnemies function, the gameTime.ElapsedGameTime.Millisceonds call always seems to return 16. This seems strange! 

![image](https://cloud.githubusercontent.com/assets/13476086/8968146/69a652e8-3633-11e5-82d6-034a96c38ebd.png)

**Comment by galadran** [comment]
From investigating gameTime further. It seems ElapsedGameTime only measures since the last update call and is consistently 16ms (it is certainly going to be fairly deterministic on any given machine). I think changing to use TotalGameTime would make more sense for the perspective of a random situation?? 

Another use of TotalGameTime could be to slowly increase the number of enemy ships attacking to force a resolution (defeat or completion) to a given scenario? 
![image](https://cloud.githubusercontent.com/assets/13476086/8968654/851a8f00-3636-11e5-97a9-bcc916b83bcb.png)

**Comment by AlanWills** [comment]
Yeah the ElapsedTime is since the last update call.  What you suggested is
actually a much better plan idea from a gameplay point of view, but the
exact numbers using the TotalGameTime would be more complicated.  I.e. I
still want random enemies appearing at the start, but obviously scaling it
depending on how long the mission goes on is a nice idea.  However, there's
probably a magic constant (or maybe a multiplier and a addition offset
constant) we're gonna need to work out to give a balanced spawning
frequency.

Basically, you're idea is great, but at the time I couldn't be bothered to
investigate the maths experimentally.  You are MORE than welcome however to
tweak things as you see fit, because the ideas you're saying are all great.
